pysolarfocus/components/heat_pump.py

```diff
@@ -11,13 +11,14 @@ def __init__(self) -> None:
         self.compressor_speed = DataValue(address=3)
         self.evu_lock_active = DataValue(address=4,type=DataTypes.UINT)
         self.defrost_active = DataValue(address=5,type=DataTypes.UINT)
-        self.boilder_charge = DataValue(address=6, type=DataTypes.UINT)
+        self.boiler_charge = DataValue(address=6, type=DataTypes.UINT)
         self.thermal_energy_total = DataValue(address=7,count=2,multiplier=0.001)
         self.thermal_energy_drinking_water = DataValue(address=9,count=2,multiplier=0.001)
         self.thermal_energy_heating = DataValue(address=11,count=2,multiplier=0.001)
         self.electrical_energy_total = DataValue(address=13,count=2,multiplier=0.001)
         self.electrical_energy_drinking_water = DataValue(address=15,count=2,multiplier=0.001)
-        self.electrical_energy_heating = DataValue(address=17,count=2,multiplier=0.001)
+        # spelling error kept for compatibility with HA-integraiton
+        self.eletrical_energy_heating = DataValue(address=17,count=2,multiplier=0.001) 
```

Comment:
Is there any way to migrate the old values to the new property name without spelling errors?

Reply:
I've fixed all spelling errors but this, as this one is probably used within energy dashboards and statistics.
To the best of my knowledge, there is no nice way to migrate this, but letting each user fix it themselves by migrating the data e.g, in their InfluxDB.

I'm still hesitating if it is worth the spelling mistake ...

Reply:
I decided to go with your proposal. -> there is a way to manually migrate it per user: https://community.home-assistant.io/t/migrate-energy-statistics-from-one-entity-to-another/405863/2